docs/blocks/custom-fluids.md

```diff
@@ -0,0 +1,851 @@
+              "fluids:z": 1
+            }
+          },
+          {
+            "condition": "query.block_neighbor_has_any_tag(1, 0, 0, 'template_full')",
+            "set_block_property": {
+              "fluids:x": 1
+            }
+          },
+          {
+            "condition": "query.block_neighbor_has_any_tag(-1, 0, 0, 'template_full')",
+            "set_block_property": {
+              "fluids:nx": 1
+            }
+          },
+          {
+            "condition": "query.block_neighbor_has_any_tag(0, 1, 0, 'template_full')",
+            "set_block_property": {
+              "fluids:top": 1
+            }
+          },
+          {
+            "condition": "query.block_neighbor_has_any_tag(0, -1, 0, 'template_full')",
+            "set_block_property": {
+              "fluids:bottom": 1
+            }
+          },
+
+          {
+            "condition": "!query.block_neighbor_has_any_tag(0, 0, -1, 'template_full')",
+            "set_block_property": {
+              "fluids:nz": 0
+            }
+          },
+          {
+            "condition": "!query.block_neighbor_has_any_tag(0, 0, 1, 'template_full')",
+            "set_block_property": {
+              "fluids:z": 0
+            }
+          },
+          {
+            "condition": "!query.block_neighbor_has_any_tag(1, 0, 0, 'template_full')",
+            "set_block_property": {
+              "fluids:x": 0
+            }
+          },
+          {
+            "condition": "!query.block_neighbor_has_any_tag(-1, 0, 0, 'template_full')",
+            "set_block_property": {
+              "fluids:nx": 0
+            }
+          },
+          {
+            "condition": "!query.block_neighbor_has_any_tag(0, 1, 0, 'template_full')",
+            "set_block_property": {
+              "fluids:top": 0
+            }
+          },
+          {
+            "condition": "!query.block_neighbor_has_any_tag(0, -1, 0, 'template_full')",
+            "set_block_property": {
+              "fluids:bottom": 0
+            }
+          },
+
+          {
+            "condition": "query.block_neighbor_has_any_tag(0, 1, 0, 'template')",
+            "run_command": {
+              "target": "self",
+              "command": [ "execute unless block ~ ~-1 ~ air unless block ~ ~-1 ~ fluids:fluid_template unless block ~ ~-1 ~ fluids:fluid_template1 unless block ~ ~-1 ~ fluids:fluid_template2 unless block ~ ~-1 ~ fluids:fluid_template3 unless block ~ ~-1 ~ fluids:fluid_template_down run setblock ~ ~ ~ fluids:fluid_template_down" ]
+            }
+          }
+        ]
+      }
+    }
+  }
+}
+```
+
+</Spoiler>
+
+## Fluid Bucket
+
+To pickup or place your custom fluid you need a custom bucket item. Although any item can pickup the fluid, your fluid can be customized to require this custom bucket. Below is the JSON for the custom bucket. To replicate the json, copy and quick replace `template` with your fluid's name.
+
+<CodeHeader>BH\items\template_bucket.json</CodeHeader>
```

Comment:
Please make the codeheader follow the same style guide, `BP/items/filename.json`